templates/src/csharp/Grpc.Auth/project.json.template

```diff
@@ -34,9 +34,9 @@
           "net45"
         ],
         "dependencies": {
-          "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc2-24027",
```

Comment:
Since we still work with the earlier versions of these libraries, wondering why we update them, just for their internal changes?